backend/Epicenter.Domain/Services/PlateService.cs

```diff
@@ -63,11 +65,19 @@ private async Task<List<RecognizedObject>> GetIdentifiedPlatesAsync(string base6
                         Message = plate.NumberPlate,
                         LastSeen = timestamp.DateTime.GetFormattedDateAndTime()
                     });
-                    _timestampRepository.Add(new Timestamp()
+                    if (seenBefore && timestamp.DateTime > DateTime.UtcNow.ToUTC2().AddMinutes(-1))
```

Comment:
DRY (don't repeat yourself) I see the same code as in faceservice, try to come with up idea on where you can put this code instead of repeating it